ironfish/src/mining/poolMiner.ts

```diff
@@ -35,6 +35,7 @@ export class MiningPoolMiner {
     publicAddress: string
     host: string
     port: number
+    minerName: string
```

Comment:
If the variable belongs to a thing, you don't need to prefix it with the name of that thing. For example, `miner.minerName` is redundant.